### DISCUSSION TITLE
Auto Download Export Xlsx file

### DISCUSSION THREAD

**Discussion by paingchan** [filamentphp/filament#15158] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

`Actions\ExportAction::make()
            ->color('info')
            ->label('Export Excel')
            ->icon('heroicon-o-arrow-down-tray')
            ->exporter(BookBuyerExporter::class)
            ->formats([
                ExportFormat::Xlsx,
            ]),` can help me  " i want to auto download .xlsx file without check noti ?

**Comment by untillnesss** [comment]
Can anyone answer?
